woocommerce/woocommerce
# set WOOCOMMERCE_BLOCKS_PHASE to 1 for the production build

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR is a POC. The current implementation is a raw approach, but it works. We should find a better approach and take care of other points listed in #43073.

Closes #43073 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a product build with the command `pnpm run --filter="@woocommerce/plugin-woocommerce" build:zip``
2. Install the zip.
3. Create a new post.
4. Open the console and print the variable `window.wcSettings.wcBlocksConfig`
5. Ensure that the `buildPhase` is set to 1.

![image](https://github.com/woocommerce/woocommerce/assets/4463174/1301206b-ff2a-4fe4-9d25-e758cc2a3cbf)




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Set WOOCOMMERCE_BLOCKS_PHASE to 1 for the production build.
</details>
